indexP2.js

```diff
@@ -0,0 +1,55 @@
+let questionPool = [];
+let displayquestion = [];
+let answer=[];
+let num=0;
+
+document.addEventListener("DOMContentLoaded", ()=>{
+    axios.get("http://localhost:3000").then(res=>{
+        questionPool = res.data;
+        shuffle(questionPool);
+        getQuestion();
+        showonHTML()
+    })
+
+    let btn =document.querySelector("button")
+    btn.addEventListener("click",event=>{
+        let sel =document.querySelector("select")
+        if(sel.selectedOptions[0].value===displayquestion.correct_answer){
+        if(!questionPool){
+            alert("you pass All Question, press ok to restart")
+            location.reload();
```

Comment:
Usually we don't want to reload the page, we always aim to not have the page reloaded